src/engines/html5/html5.js

```diff
@@ -1016,6 +1016,7 @@ export default class Html5 extends FakeEventTarget implements IEngine {
   _onCueChange(e: FakeEvent): void {
     let textTrack: TextTrack = e.currentTarget;
     let activeCues: Array<Cue> = [];
+    if (!textTrack || !textTrack.activeCues) return;
```

Comment:
how can this happen? if you get the event it means that textTrack must exist.
If activeCues is empty we still want the event to be dispatched.

Reply:
Right.
activeCue is null and it's not iterable, but it should fire the evet. i'll fix it

Reply:
active cue is not null, it is an empty array, unless the text track is disabled and then it is null but then you shouldn't get the event.

Reply:
it happened multiple times for me.
it occurs as part of detach sequence, it's not always happened it's depend on timing of detach request.

Reply:
So this seems like a regression from the current change, please check.

Reply:
I have checked it multiple time and I can't reproduce.
I guess depend on the timing of detach and cues.